corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -325,7 +325,7 @@ var COMMCAREHQ = (function () {
 function setUpIeWarning() {
     'use strict';
     var $warning;
-    if ($.browser.msie) {
+    if (/msie/i.test(navigator.userAgent) || /windows/i.test(navigator.userAgent)) {
```

Comment:
is there a reason why this had to change? $.browser's deprecation only happens for Bootstrap3 portions of the site, and currently no B3 templates to my knowledge are using main.js

Really, this should be handled by importing a separate js file from I.E.'s comment versioning syntax and activating the alert from there, not from within a framework